README.undock.md

```diff
@@ -48,30 +48,48 @@ Note how the current code uses the `winuwp` OS variant to enable this. This is b
 
 #### Build
 
-Note that there is currently no difference between Desktop and UWP builds. More precisely, the library produced is only useable on Desktop at the moment, as it uses some non-UWP APIs.
-
-NB: Do **not** use `target_os="winuwp"`, this is the old legacy target that is being removed, and merged with the regular Desktop one (`target_os="win"`).
+In order to build `webrtc.lib`, run:
 
 ```cmd
 cd src
-gn args out\debug_arm64
+gn args out\$(configuration)_$(cpu)[_win32]

```

Comment:
nit: Ideally we'd want something symmetrical, so a `_uwp` suffix. Can fix later.

Reply:
Ideally we'd want build.ps1 to do this automatically I'd say!